packages/showcases/core/csf/components/indicators/tag.csf.js

```diff
@@ -57,4 +57,13 @@ export const argTypes = {
       options: ['', ...Object.keys(vitamixIconsList)],
```

Comment:
Please update the css storybook at `packages/showcases/css/stories/components/indicators/tag/examples`.
- Add `vtmn-tag_size--medium` on each existing stories 
- Create a new bloc with the new class like 
```html
<div class="block">
  <span class="vtmn-tag vtmn-tag_variant--accent vtmn-tag_size--small"
    >Accent Tag</span
  >

  <span
    class="vtmn-tag vtmn-tag_variant--decorative_gravel vtmn-tag_size--small"
    >Gravel</span
  >

  <a href="" class="vtmn-tag vtmn-tag_variant--alert vtmn-tag_size--small"
    ><span class="vtmx-leaf-fill" role="presentation"></span>Interactive Tag
    with icon</a
  >
</div>
```

Reply:
Could you check again ?